components/LocationComparisonSection/LocationComparisonBanner.tsx

```diff
@@ -1,20 +1,28 @@
-import React, { CSSProperties, FunctionComponent, useState } from 'react';
+import React, { CSSProperties, FunctionComponent, useEffect, useState } from 'react';
 import { SpotlightLocation } from '../../utils';
 import { Button } from '../Button';
 import { ButtonBanner } from '../ButtonBanner';
 import { LocationSelectionBanner } from '../LocationSelectionBanner';
 import { SpotlightBanner } from '../SpotlightBanner';
 import { TagList, TagListItem } from '../Tags';
+import { Alert } from '../Alert';
+import { Icon } from '../Icon';
 
 interface ComparisonWrapperProps {
   countryName: string;
   countryCode: string;
   onCompare?: (locations: SpotlightLocation[]) => void;
+  queryLocation?: SpotlightLocation[];
 }
 
 const LocationComparisonBanner: FunctionComponent<ComparisonWrapperProps> = props => {
   const [addLocation, setAddLocation] = useState(false);
   const [locations, setLocations] = useState<SpotlightLocation[]>([]);
+  useEffect(() => {
+    if (props.queryLocation) {
+      setLocations(props.queryLocation);
+    }
+  }, []);
```

Comment:
why not add it as a default on line 20 instead of that empty array?

Reply:
Updated